tests/e2e/ansible/group_vars/all

```diff
@@ -5,6 +5,7 @@ build_pkgs:
   ubuntu:
     - make
     - gcc
+    - qemu-user-static
```

Comment:
Instead of using system packages, as IIRC, it was a problem with CentOS, could we use something like https://github.com/multiarch/qemu-user-static ?

Reply:
Oh, I'm dumb, it's already used. <3